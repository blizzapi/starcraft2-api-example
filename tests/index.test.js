const request = require('supertest');
const app = require('../src/app');

describe('/', () => {
  it('should return 200', async () => {
    const response = await request(app).get('/');
    expect(response.statusCode).toBe(200);
  });

  it('should match snapshot', async () => {
    const response = await request(app).get('/');
    expect(response.body).toMatchSnapshot();
  });
})