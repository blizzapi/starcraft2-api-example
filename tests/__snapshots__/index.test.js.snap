// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/ should match snapshot 1`] = `
Object {
  "message": "hello world!",
}
`;

exports[`/queryGrandmasterLeaderboard/1 should match snapshot 1`] = `
Object {
  "data": "sample data for queryGrandmasterLeaderboard",
}
`;

exports[`/queryLadderSummary/1/1/123456 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLadderSummary",
}
`;

exports[`/queryLeagueData/1/37/201/0/6 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLeagueData",
}
`;

exports[`/queryLegacyAchievements/1 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLegacyAchievements",
}
`;

exports[`/queryLegacyLadder/1/234567 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLegacyLadder",
}
`;

exports[`/queryLegacyLadders/1/1/123456 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLegacyLadders",
}
`;

exports[`/queryLegacyMatchHistory/1/1/123456 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLegacyMatchHistory",
}
`;

exports[`/queryLegacyProfile/1/1/123456 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLegacyProfile",
}
`;

exports[`/queryLegacyRewards/1 should match snapshot 1`] = `
Object {
  "data": "sample data for queryLegacyRewards",
}
`;

exports[`/queryPlayerAccount/1/123 should match snapshot 1`] = `
Object {
  "data": "sample data for queryPlayerAccount",
}
`;

exports[`/queryPlayerLadder/1/1/123456/234567 should match snapshot 1`] = `
Object {
  "data": "sample data for queryPlayerLadder",
}
`;

exports[`/queryProfile/1/1/123456 should match snapshot 1`] = `
Object {
  "data": "sample data for queryProfile",
}
`;

exports[`/queryProfileMetadata/1/1/123456 should match snapshot 1`] = `
Object {
  "data": "sample data for queryProfileMetadata",
}
`;

exports[`/querySeason/1 should match snapshot 1`] = `
Object {
  "data": "sample data for querySeason",
}
`;

exports[`/queryStaticProfileData/1 should match snapshot 1`] = `
Object {
  "data": "sample data for queryStaticProfileData",
}
`;
